controllers/constant/odlm.go

```diff
@@ -2136,6 +2176,14 @@ spec:
               - name: ibm-im-operator
             registry: common-service
   - name: ibm-zen-operator
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
```

Comment:
```suggestion
        namespace: "{{ .ServicesNs }}"
```
For the namespace of `meta-api-deploy` in only `ibm-zen-operator` under CPFS v3, it would use serviceNs
typo fix in: https://github.com/IBM/ibm-common-service-operator/pull/2282/files